src/vllm_tgis_adapter/__main__.py

```diff
@@ -33,14 +274,12 @@ async def _force_log() -> None:
 
     server = await start_grpc_server(async_llm_engine, args)
 
-    try:
-        while True:
-            await asyncio.sleep(60)
-    except asyncio.CancelledError:
-        logger.info("Gracefully stopping gRPC server")
-        await server.stop(30)  # TODO configurable grace
-        await server.wait_for_termination()
-        logger.info("gRPC server stopped")
+    yield
```

Comment:
I did not notice this before merging. Note that this makes it impossible to gracefully stop the grpc server and could (will?) result in the process hanging on ctrl-c